.github/workflows/llext.yml

```
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [mtl, lnl]
+        platforms: [mtl]
```

Create an empty, placeholder file `sof/app/overlays/mtl/module_overlay.conf` and then use `--overlay=sof/app/overlays/${{ matrix.platform }}/module_overlay.conf`

Keep the field name singular: it's a field name not a vector name.

```suggestion
        platform: [mtl, lnl]
```

---

@marc-hb ehm, I guess you meant
```
+        platform: [mtl]
```
right? Can do, sure, but TBH I find a plural form with an array logical enough, even if **currently** that array happens to only have one member